src/reducers/ui.js

```diff
@@ -24,6 +24,29 @@ const defaultState = new Immutable.Map().
   set('topBar', new Immutable.Map({openMenu: null})).
   set('lastRefreshTimestamp', null);
 
+// i.e. restore to before editors were resized
```

Comment:
I’m generally not a fan of most “explanatory” comments—it’s very easy for them to diverge from the code they seek to explain, and if variables/functions/classes are named carefully, I don’t think comments are usually necessary. In this case I think the function names `restoreDefaultColumnFlex` etc. are entirely self-documenting!